controllers/suite_test.go

```diff
@@ -94,69 +100,62 @@ var (
 )
 
 var (
-	testRegistryClient *helmreg.Client
-	testRegistryserver *RegistryClientTestServer
-)
-
-var (
-	testWorkspaceDir         = "registry-test"
-	testHtpasswdFileBasename = "authtest.htpasswd"
-	testUsername             = "myuser"
-	testPassword             = "mypass"
+	testRegistryServer *registryClientTestServer
 )
 
 func init() {
 	rand.Seed(time.Now().UnixNano())
 }
 
-type RegistryClientTestServer struct {
-	Out                io.Writer
-	DockerRegistryHost string
-	WorkspaceDir       string
-	RegistryClient     *helmreg.Client
+type registryClientTestServer struct {
+	out                io.Writer
+	dockerRegistryHost string
```

Comment:
This is okay, but since there's another change that you'd make, how about also changing this to be more generic, like just `registryHost`, remove the details about the registry.